src/main/java/seedu/address/logic/commands/EditBlockCommand.java

```diff
@@ -0,0 +1,88 @@
+package seedu.address.logic.commands;
+
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.UserPrefs;
+
+/**
+ * Edits the range of block letter.
+ */
+public class EditBlockCommand extends Command {
+
+    public static final String COMMAND_WORD = "edit-block-range";
+
+    public static final String COMMAND_NOTES = "Notes: Please restart the application for the block "
+        + "settings changes to take effect. This command will erase the existing data "
+        + "if the resident's block letter is not longer valid after updating the block settings.";
```

Comment:
typo here, 'not longer' 
can consider changing the "erase the existing data" to "reset it to default settings"